packages/ui/origin-ui-core/src/components/certificates/certificateMenuCreator.tsx

```diff
@@ -79,6 +80,13 @@ export const certificatesMenuCreator = (user: IUser): ICertificateMenuItem[] =>
             show: userIsActiveAndPartOfOrg,
             features: [OriginFeature.Certificates, OriginFeature.Buyer]
         },
+        {
+            key: 'inbox2',
+            label: 'inbox2',
```

Comment:
label should be a path to localization property. 
Also `inbox2` should be changed to something like `Exchange Inbox`